src/DropNCrop.js

```diff
@@ -127,65 +129,71 @@ class DropNCrop extends Component {
 
     return (
       <div className={classNames(imageCropper)}>
-        {this.state.src ? (
-          <Cropper
-            ref={(input) => { this.cropperRef = input; }}
-            src={this.state.src}
-            style={{
-              height: this.props.canvasHeight,
-              width: this.props.canvasWidth,
-            }}
-            cropend={this.onCrop} // Use cropend to reduce setState lag while cropping
-            {...this.props.cropperOptions}
-          />
-        ) : (
-          <Dropzone
-            className="dropzone"
-            activeClassName="dropzone--active"
-            onDrop={this.onDrop}
-            style={{
-              height: this.props.canvasHeight,
-              width: this.props.canvasWidth,
-            }}
-          >
-            <div key="dropzone-instructions">
-              {!this.props.instructions ? (
-                <div className="dropzone-instructions">
-                  <div className="dropzone-instructions--main">Drag-n-drop a file or click to add an image</div>
-                  <div className="dropzone-instructions--sub">Accepted file types: .jpg, .jpeg, .png</div>
-                  <div className="dropzone-instructions--sub">Max file size: {bytesToSize(this.props.maxFileSize)}</div>
-                </div>
-              ) : this.props.instructions}
-            </div>
-            {this.state.error ? (
-              <div
-                key="dropzone-validation"
-                className="dropzone-validation"
-              >
-                {this.state.error}
+        {this.state.src
+          ? <Cropper
+              ref={input => {
+                this.cropperRef = input;
+              }}
+              src={this.state.src}
+              style={{
+                height: this.props.canvasHeight,
+                width: this.props.canvasWidth,
+              }}
+              cropend={this.onCrop} // Use cropend to reduce setState lag while cropping
+              {...this.props.cropperOptions}
+            />
+          : <Dropzone
+              className="dropzone"
+              activeClassName="dropzone--active"
+              onDrop={this.onDrop}
+              style={{
+                height: this.props.canvasHeight,
+                width: this.props.canvasWidth,
+              }}
+            >
+              <div key="dropzone-instructions">
+                {!this.props.instructions
+                  ? <div className="dropzone-instructions">
+                      <div className="dropzone-instructions--main">
+                        Drag-n-drop a file or click to add an image
+                      </div>
+                      <div className="dropzone-instructions--sub">
+                        Accepted file types: .jpg, .jpeg, .png
```

Comment:
This should iterate over the `types` passed in via `allowedFileTypes` prop and print them